shared/js/background.js

```diff
@@ -69,9 +69,16 @@ function Background() {
   });
 
   chrome.runtime.onInstalled.addListener(function(details) {
-    // only run the following section on install
+    // only run the following section on install and on update
```

Comment:
doesn't this mean it will show on upgrade? we do not want to do that at this time.

Reply:
I think you'll want to do 
`if (details.reason.match(/install/) { 
  settings.ready().then( () => { 
     settings.updateSetting( 'hasSeenPostInstall', true)
     chrome.tabs.update( ... )
}`

Double-check that everything above is correct but that's a rough idea of what should be happening here.

Reply:
(Sorry for formatting above)